aom/src/main/java/com/nedap/archie/query/AOMPathQuery.java

```diff
@@ -25,11 +25,20 @@
  */
 public class AOMPathQuery {
 
-    private List<PathSegment> pathSegments = new ArrayList<>();
+    private final List<PathSegment> pathSegments;
+
+    /** If true, extend the search through C_COMPLEX_OBJECT_PROXY objects by looking up the replacement first.*/
+    private final boolean findThroughCComplexObjectProxies;
```

Comment:
So when should or shouldn't the proxy be replaced when doing an apathQuery? Seems to me it should always replace it?